trillian/log_server/trampoline.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"syscall"
```

Comment:
Unfortunately this isn't really supposed to be used anymore, but there's no other way to get `execve(2)` it seems, even with the `golang.org/x/sys` package recommended as a replacement.